beacon-chain/node/node_test.go

```diff
@@ -47,3 +48,31 @@ func TestNodeClose_OK(t *testing.T) {
 		t.Log(err)
 	}
 }
+func TestBootStrapNodeFile(t *testing.T) {
+	if _, err := os.Stat("sampleNodes.enr"); os.IsExist(err) {
+		t.Fatalf("File sampleNodes.enr already exists. Aborting test so that file is not altered.")
+	}
+
+	defer func() {
+		err := os.Remove("sampleNodes.enr")
+		if err != nil {
+			t.Error("Could not delete temporary file sampleNodes.enr")
+		}
+	}()
+
+	sampleNode0 := "- enr:-Ku4QMKVC_MowDsmEa20d5uGjrChI0h8_KsKXDmgVQbIbngZV0i" +
+		"dV6_RL7fEtZGo-kTNZ5o7_EJI_vCPJ6scrhwX0Z4Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpD" +
+		"1pf1CAAAAAP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQJxCnE6v_x2ekgY_uo" +
+		"E1rtwzvGy40mq9eD66XfHPBWgIIN1ZHCCD6A"
+	sampleNode1 := "- enr:-TESTNODE2"
+	sampleNode2 := "- enr:-TESTNODE3"
```

Comment:
sampleNode0 is just prysm's default bootstrap node.  I chose some bland text for node #2 and #3 but perhaps it should be something else.  I didn't know if I should throw in any other nodes (such as from my active beacon-chain) as that might unnecessarily list actual nodes.